src/components/cylc/Drawer.vue

```diff
@@ -68,11 +67,14 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 </template>
 
 <script>
```

Comment:
I spent a bit of time yesterday trying to make this into <script setup> but was struggling with a few bits

`export default {
  components: {
    Workflows,
    'c-header': Header
  },
  icons: {
      mdiHome,
      mdiGraphql,
    },
  mode: import.meta.env.MODE,
  version: pkg.version,
}`

Reply:
There is no need to specify components in `<script setup>`, just import them e.g. `import CHeader from '@/components/cylc/Header.vue'